critiquebrainz/db/review.py

```diff
@@ -179,7 +183,7 @@ def get_count(*, is_draft=False, is_hidden=False):
         return result.fetchone()[0]
 
 
-def update(review_id, *, drafted, text, license_id=None, language=None, is_draft=None):
+def update(review_id, *, drafted, text, rating, license_id=None, language=None, is_draft=None):
```

Comment:
There is a problem here. The idea that we had is that review is one of the following:

* text *(how it is right now)*
* rating
* text + rating

Looking at the signature of this function it seems like both text and review are mandatory (arguments). First of all, this is not going to work with the current implementation that depends on this function. It doesn't provide rating, obviously, since that part is missing. But even without accounting for that, in the future it needs to support different variations of reviews (as I mentioned before).

What I'd do here instead is make both `text` and `rating` arguments optional, then check that at least one of those is provided.